demos/cpp/.vscode/launch.json

```diff
@@ -23,7 +28,7 @@
                 }
             ],
             "preLaunchTask": "g++ build active file",
-            "miDebuggerPath": "/usr/bin/gdb"
+            "miDebuggerPath": "C:\\Program Files\\mingw-w64\\x86_64-8.1.0-posix-seh-rt_v6-rev0\\mingw64\\bin\\gdb.exe"
```

Comment:
I think this is very likely to break on someone elses machine!

Reply:
I have tried to keep the installation of mingw-w64 on windows as default as possible.

Reply:
I would just drop the miDebuggerPath all together, gdb should be ideally picked from the PATH, anything will be case dependant